infer/environment.cc

```diff
@@ -815,7 +815,6 @@ core::TypePtr Environment::processBinding(core::Context ctx, cfg::Binding &bind,
             },
             [&](cfg::Alias *a) {
                 core::SymbolRef symbol = a->what.data(ctx)->dealias(ctx);
-                lspQueryMatch = lspQueryMatch || lspQuery.matchesSymbol(symbol);
```

Comment:
Yaay, Alias is an implementation detail of cfg&infer and I'm happy that LSP integration no longer depends on it!